### ISSUE TITLE
Install as an OS?

### ISSUE THREAD

**Issue by Dvalin21** [IceWhaleTech/CasaOS#757]
**Is your feature request related to a problem? Please describe.**
Not related to a problem, but possibly something others have thought about, but could make it easier (Its not hard to install for some, but for those who will eventually want to try this out and don't have the technical know how) to install. With it being installed as a whole as a OS, some specific features could be added i.e. hibernation already disabled, for those who have wol enabled in bios, feature either already enabled or able to be enabled through casaos gui, ssh setup happens during the install and just have a section to enable or disable and maybe being able to change the port.

Note: By no means am I a dev, however I do like the project that you guys are doing and if an idea could help make it better, I would love to see it happen.

**Comment by jerrykuku** [comment]
You mean it's better to have a complete installation rather than installing on an existing system via an install script, right?

**Comment by Dvalin21** [comment]
Correct, I guess I could have worded it differently. Thanks

**Comment by jerrykuku** [comment]
I agree with you. We are trying to do the same, but the process will take longer and we will release it once we are ready.

**Comment by Dvalin21** [comment]
Thanks for the response. I look forward to seeing all the progress!!!

**Comment by ws245909346** [comment]
老竭力大佬

**Comment by ETWang1991** [comment]
A disk merge feature similar to raid has been released in version 0.4.1

**Comment by Dvalin21** [comment]
Thanks ETWang1991, is there documentation on the website for this? Thanks
in advance.

**Comment by udance4ever** [comment]
isn’t CasaOS closer to what Springboard is on iOS? (excuse my ignorance, I’m just now learning what these components do)

meaning CasaOS depends on an underlying operating system to perform all the low level functions and system calls?

I assume this means if CasaOS were offered standalone (which I think is a great idea), it would be bundled and packaged with a some baseline OS (eg. Ubuntu Server)
